node/api/api.go

```diff
@@ -29,6 +32,25 @@ type Error struct {
 	// be valid or invalid depending on the current state of a module.
 }
 
+type PaginationRequest struct {
```

Comment:
I think we don't need that struct at all. We can just return multiple return values in the methods that use it.

Reply:
Btw all exported structs and methods need a comment.